extension/src/prototypes/view/containers/ScreenSpaceSelection.tsx

```diff
@@ -16,30 +14,19 @@ const EVENTS_ON_SELECT_TOOL = {
   select: true,
 };
 
-const EVENTS_ON_STORY_TOOL = {
-  point: true,
-  rectangle: false,
-  select: false,
-};
-
 export type ScreenSpaceSelectionProps = Omit<PlateauScreenSpaceSelectionProps, "disabled">;
 
-const isFeatureStoryCapture = (f: PickedFeature | undefined) =>
-  f?.properties && !!f.properties[STORY_MARKER_ID_PROPERTY];
-
 export const ScreenSpaceSelection: FC<ScreenSpaceSelectionProps> = props => {
   const tool = useAtomValue(toolAtom);
   const theme = useTheme();
   const isMobile = useMediaQuery(theme.breakpoints.down("mobile"));
   return (
     <PlateauScreenSpaceSelection
       {...props}
-      disabled={tool?.type !== "select" && tool?.type !== "story"}
+      disabled={tool?.type !== "select"}
       allowClickWhenDisabled={isMobile}
-      filterSelectedFeature={tool?.type === "story" ? isFeatureStoryCapture : undefined}
-      allowedEvents={
-        isMobile || tool?.type === "select" ? EVENTS_ON_SELECT_TOOL : EVENTS_ON_STORY_TOOL
-      }
+      filterSelectedFeature={undefined}
+      allowedEvents={isMobile || tool?.type === "select" ? EVENTS_ON_SELECT_TOOL : undefined}
```

Comment:
Please help check this part since story is no longer related with any screen space selection.